cdap-watchdog/src/main/java/co/cask/cdap/metrics/MetricsConstants.java

```diff
@@ -48,8 +48,8 @@
   public static final int DEFAULT_TIME_SERIES_TABLE_ROLL_TIME = 3600;
   public static final long DEFAULT_RETENTION_HOURS = 2;
 
-  public static final long METRICS_HOUR_RESOLUTION_CUTOFF = 3610;
-  public static final long METRICS_MINUTE_RESOLUTION_CUTOFF = 610;
+  public static final long METRICS_HOUR_RESOLUTION_CUTOFF = 36000;
```

Comment:
no need to have METRICS prefix. Also might wanna call it MAX_HOUR_RESOLUTION_QUERY_INTERVAL or smth